### ISSUE TITLE
Communication Error: builder error: relative URL without a base

### ISSUE THREAD

**Issue by benjaminleonard** [oxidecomputer/oxide.rs#250]
I get the following error on the CLI:

```
error
Communication Error: builder error: relative URL without a base
```

With any command (other than `oxide version` or `oxide auth`). Auth works fine and I've cleared `hosts.toml`. Doesn't change the error.

hosts.toml
```
["https://oxide.sys.rack2.eng.oxide.computer/"]
token = "oxide-token-c81816ad825af4f96cd7b9480ea7a549589b374d"
user = "a7399623-06b5-4e44-9620-80485e5ca0e6"
```

**Comment by askfongjojo** [comment]
@benjaminleonard - This probably has to do with the OXIDE_HOST env variable. You might have set it as a hostname instead of http endpoint (which is an easily confused thing people have brought up previously).
```
$ OXIDE_HOST=oxide.sys.rack2.eng.oxide.computer oxide current-user view
error
Communication Error: builder error: relative URL without a base

$ OXIDE_HOST=https://oxide.sys.rack2.eng.oxide.computer oxide current-user view
success
CurrentUser {
    display_name: "angela@oxidecomputer.com",
    id: d0ff493f-a09d-47e2-b266-70db72a495a8,
    silo_id: d9752658-b163-421c-9f5c-516099fdc44f,
    silo_name: Name(
        "oxide",
    ),
}
```

**Comment by ahl** [comment]
I believe this is no longer relevant after #727, but feel free to re-open if it's still an issue.
